roles/prometheus/templates/etc/systemd/system/alertmanager.service.j2

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Alertmanager
+After=network.target
+
+[Service]
+Type=simple
+PIDFile=/var/run/alertmanager.pid
+ExecReload=/bin/kill -HUP $MAINPID
+ExecStart=/opt/prometheus/current/alertmanager-{{ alertmanager_version }}.linux-amd64/alertmanager -config.file=/opt/prometheus/current/alertmanager-{{ alertmanager_version }}.linux-amd64/simple.yml
+Restart=always
+SyslogIdentifier=alertmanager]
```

Comment:
Trailing ]

You're missing the other settings that'd send the logs to syslog which would make this useful